src/patternfly/components/Radio/radio.scss

```diff
@@ -38,7 +38,7 @@
 
 .#{$radio}__input {
   height: var(--#{$radio}__input--Height);
-  margin-top: var(--#{$radio}__input--MarginTop);
+  transform: translate(var(--#{$radio}__input--TranslateY));
```

Comment:
This should be `translateY`
Also, I'd suggest adding the same comment here to explain why we must manually nudge this with translate.